app/server.py

```diff
@@ -1,11 +1,13 @@
 import os
 import jinja2
+import quandl
+import psycopg2
 from klein import Klein
 
 
 app = Klein()
 app.templates = jinja2.Environment(loader=jinja2.FileSystemLoader('templates'))
-
+connection = psycopg2.connect(database='snp', user='Jessica', host='localhost')
```

Comment:
I know I said to put this here for now until we see a reason to move it.  I think maybe this line should go in the db module.  If it's going to be an argument to every function defined in the db module, it might as well live there.